modules/tensor_mechanics/src/materials/ComputeInterfaceStress.C

```diff
@@ -18,51 +18,75 @@ validParams<ComputeInterfaceStress>()
   params.addClassDescription(
       "Stress in the plane of an interface defined by the gradient of an order parameter");
   params.addCoupledVar("v",
-                       "Order parameter that defines the interface. The interface is the region "
+                       "Order parameters that defines the interface. The interface is the region "
                        "where the gradient of this order parameter is non-zero.");
-  params.addRequiredParam<Real>("stress", "Planar stress");
-  params.addRangeCheckedParam<Real>("op_range",
-                                    1.0,
-                                    "op_range > 0.0",
-                                    "Range over which order parameters change across an "
-                                    "interface. By default order parameters are assumed to "
-                                    "vary from 0 to 1");
+  params.addRequiredParam<std::vector<Real>>("stress", "Planar stress");
+  params.addRangeCheckedParam<std::vector<Real>>(
```

Comment:
This part is also unclear in the parameter description.  This parameter is now a vector as well, so you can have multiple ranges?

Reply:
I had to look this up myself but `paramname` is substituted with each vector entry (i.e. teh check applies to all entries), `paramname_size` checks the size of the vector, and `paramname_3` checks `vector[3]` etc.

Turns out I wrote the range checking for vector parameters myself 3.5 years ago :-D